packages/office-ui-fabric-react/src/components/ActivityItem/ActivityItem.Props.ts

```diff
@@ -0,0 +1,103 @@
+import * as React from 'react';
+import { IStyle } from '../../Styling';
+import { IPersonaProps } from 'office-ui-fabric-react/lib/Persona';
+
+// Please keep alphabetized
+export interface IActivityItemProps extends React.HTMLAttributes<HTMLElement> {
+  /**
+   * Text or elements in this array will be shown in order as the description of the activity that occurred.
+   */
+  activityDescription: Array<string | JSX.Element>;
+
+  /**
+   * Text or elements of a comment that will be shown in order under the activity description.
+   */
+  commentElements?: Array<string | JSX.Element>;
+
+  /**
+   * Used to render the main icon. Pass in an array of Persona props to automatically render a Persona group, or pass in a JSX element to render your own content, such as an Icon, here.
+   */
+  iconContents: Array<IPersonaProps> | JSX.Element;
```

Comment:
I think i'd like to split this up. Letting iconContents be an array of personas OR a JSX.Element are 2 completely different things, also there is no opportunity to access props in that returned element.

We could make this (and some of the above similar props) into iconContents and onRenderIconContents. So there is a default method for rendering, or you can pass the props in and still customize how it is rendered.